### DISCUSSION TITLE
Enquiry about the preferred structure for the project.

### DISCUSSION THREAD

**Discussion by nerfednerd** [YaliWang2019/AK-Satellite-Imagery-Wildfire-Prediction#21] [General]
Hiii @YaliWang2019, 

Since the repository is currently empty, could you please help us with a few questions about the project, which would be helpful for folks like me who want to contribute to the Alaska Wildfire Prediction Project.

1. Do you plan to use Google Earth Engine (GEE) (via earthengine-api) to handle the satellite datasets, or should we build a scraper using sentinelsat / CDS API to download raw GeoTIFFs locally?

2. Do you have a strict preference for PyTorch or TensorFlow for the CNN-LSTM model?

3. For Sentinel-2 preprocessing, do you want to implement a custom cloud-masking algorithm, or should we utilize the s2cloudless library as a starting point?

Best Regards
